rolo/client.py

```diff
@@ -56,9 +56,11 @@ def merge_environment_settings(self, url, proxies, stream, verify, *args, **kwar
 
 class SimpleRequestsClient(HttpClient):
     session: requests.Session
+    follow_redirects: bool
 
-    def __init__(self, session: requests.Session = None):
+    def __init__(self, session: requests.Session = None, follow_redirects: bool = True):
```

Comment:
since we kept the parameter specific to requests, i would have kept the original name of `allow_redirects`, so it's consistent with the requests library :-) though `follow_redirects` is the better name to generalize if we added it to _all_ client implementations